frontend/src/views/Administration.vue

```diff
@@ -557,7 +557,7 @@ export default {
       margin: 0px;
     }
   }
-  .wrap-text {
+  .wrap-title-text {
```

Comment:
Same as above. Since the style is scoped do not rename.

Reply:
same as my comment above. Will revert it.